examples/dig/README.md

```diff
@@ -0,0 +1,20 @@
+# Dependency Injection Graph example
+
+This example illustrates how `package go.uber.org/fx/dig` can be used to inject
+a dependency.
+
+`struct HelloHandler` exploses a dependency on `hello.Sayer` but does not
+provide any guidance of when or where to get it. On the other side package sayer
```

Comment:
"On the other side package sayer" is awkward. How about "However, package sayer injects…"? I'm expecting to see something about package sayer providing guidance in contrast to HelloHandler not providing any guidance. Not sure what the point is here.